source/ShopPanel.cpp

```diff
@@ -1413,6 +1467,10 @@ vector<ShopPanel::Zone>::const_iterator ShopPanel::Selected() const
 // letter of the button (or ' ' if it's not on a button).
 char ShopPanel::CheckButton(int x, int y)
 {
+	if(x > Screen::Right() - SIDEBAR_WIDTH - 380 && x < Screen::Right() - SIDEBAR_WIDTH - 320 &&
```

Comment:
The `ShopPanel::CheckButton()` function begins by ignoring all points outside the "button zone", so I had to add the find button clicking functionality to before this part. Since it's separate from the other button code, it's kind of ugly imo.

Reply:
Looks fine to me.